src/osd/PG.cc

```diff
@@ -1410,14 +1410,13 @@ bool PG::recoverable_and_ge_min_size(const vector<int> &want) const
           pool.info.is_erasure() ? shard_id_t(i) : shard_id_t::NO_SHARD));
     }
   }
-
```

Comment:
I think this kind of cleanup will pollute the commit history, which make us can not use `git show` and `git blame` to dig commit history, right? Maybe we should not introduce it~